### PR TITLE
fix: Update the jobs readme description

### PR DESCRIPTION
**Pull request by sradevski**

### CODE REVIEW

File: src/jobs/README.md [hunk 1]
```diff
@@ -35,8 +33,6 @@ A scheduled job file must export:
 - A configuration object defining the job. It has two properties:
```
Comment (shahednasser) [inline src/jobs/README.md]:
```suggestion
- A configuration object defining the job. It has three properties:
```

File: src/jobs/README.md [hunk 2]
```diff
@@ -35,8 +33,6 @@ A scheduled job file must export:
...
   - `name`: a unique name for the job.
   - `schedule`: a [cron expression](https://crontab.guru/).
+  - `numberOfExecutions`: an optional integer, specifying how many times the job will execute before being removed
 
-The `handler` is a function which takes one parameter, an `object` of type `ScheduledJobArgs` with the following properties:
-
-- `container` - a `MedusaContainer` instance which can be used to resolve services.
-- `data` - an `object` containing data passed to the job when it was scheduled. This object is passed in the `config` object.
+The `handler` is a function which takes one parameter, `container`, a `MedusaContainer` instance which can be used to resolve services.
```
Comment (shahednasser) [inline src/jobs/README.md]:
```suggestion
The `handler` is a function that accepts one parameter, `container`, which is a `MedusaContainer` instance used to resolve services.
```

**Review by shahednasser — CHANGES_REQUESTED** [review]
Thanks for updating this, wasn't aware that scheduled jobs are ready for use now so need to update the docs as well 👍🏻

Shouldn't the example code be updated as well?

**Comment by sradevski** [comment]
@shahednasser yes indeed, thanks for catching that. Updated it now

**Review by shahednasser — APPROVED** [review]
Made direct edits to code snippet, hope that's ok 😄

LGTM Thanks!

**Comment by sradevski** [comment]
Yes ofc, thanks @shahednasser

**Pull request merged**